src/snowflake/cli/plugins/object/commands.py

```diff
@@ -21,6 +21,15 @@
 ObjectArgument = typer.Argument(
     help="Type of object. For example table, procedure, streamlit.",
     case_sensitive=False,
+    show_default=False,
+)
+ObjectDefinitionArgument = typer.Argument(
+    help="""Object definition in JSON format, for example \'{"name": "my_database", "owner": "owner_role"}\',
```

Comment:
Is this a valid example?

Reply:
I've checked - "owner" is not settable field, it is ignored by request (that's why my previous check passed). I've changed it to "comment"